apps/monarch/monarch-frontend/src/app/SearchTherapists.tsx

```diff
@@ -266,9 +271,8 @@ const EditButton: React.FC<{
         if (setReload) {
           setReload(true);
         }
+        onEditClose();
```

Comment:
moving `onEditClose()` into the promise callback since sometimes the request would complete after the modal disappears, making it seem like nothing changed